entity-types/ext-coredns/tests/Metric.stg.json

```diff
@@ -0,0 +1,39 @@
+[
+  {
+    "cluster_name": "demo-june-4",
+    "collector.name": "prometheus-agent",
+    "collector.version": "2.0.0",
+    "coredns_build_info": {
+      "type": "gauge",
+      "count": 1,
+      "sum": 1,
+      "min": 1,
+      "max": 1,
+      "latest": 1
+    },
+    "eks_amazonaws_com_component": "kube-dns",
+    "entity.guid": "MTE5OTkwMzZ8RVhUfENPUkVETlN8LTg4OTM5NzYxNjUwMTcyOTQ5OTU",
+    "entity.name": "10.0.32.41:9153",
+    "entity.type": "COREDNS",
```

Comment:
if the telemetry already has the guid stamped our engine assumes it has already gone through entity synthesis so we just discard it, I'd remove these three fields as they are stamped by synthesis service